src/components/DoiMetadata/DoiMetadata.tsx

```diff
@@ -190,6 +239,8 @@ const DoiMetadata: React.FunctionComponent<Props> = ({item}) => {
         {creators()}
         {metadata()}
         {description()}
+        {registered()}
```

Comment:
On the placement of the DOI registration date, shouldn't we consider a place near the DOI display --just after the abstract seems obscured and unrelated?

Reply:
The DOI registration date is not really relevant to most users. I show it only for the cases where the publication year is much earlier. Showing the registration agency is important, but does not have to be prominent.